gcloud/storage/_helpers.py

```diff
@@ -82,7 +73,7 @@ def batch(self):
         """
         return _PropertyBatch(self)
 
-    def _reload_properties(self):
+    def reload(self):
```

Comment:
Or we can create an issue for cleaning those bits out (e.g., `git grep "return self$").